pkg/cmd/auth/refresh/refresh.go

```diff
@@ -0,0 +1,127 @@
+package refresh
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/internal/config"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/pkg/prompt"
+	"github.com/spf13/cobra"
+)
+
+type RefreshOptions struct {
+	IO     *iostreams.IOStreams
+	Config func() (config.Config, error)
+
+	Hostname string
+	Scopes   []string
+}
+
+func NewCmdRefresh(f *cmdutil.Factory, runF func(*RefreshOptions) error) *cobra.Command {
+	opts := &RefreshOptions{
+		IO:     f.IOStreams,
+		Config: f.Config,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "refresh",
+		Args:  cobra.ExactArgs(0),
+		Short: "Request new scopes for a token",
+		Long: heredoc.Doc(`Expand the permission scopes for a given host's token.
+
+			This command allows you to add additional scopes to an existing authentication token via a web
+			browser. This enables gh to access more of the GitHub API, which may be required as gh adds
+			features or as you use the gh api command. 
+
+			Unfortunately at this time there is no way to add scopes without a web browser's involvement
+			due to how GitHub authentication works.
+
+			The --hostname flag allows you to operate on a GitHub host other than github.com.
```

Comment:
Could we omit this line since it doesn't add new information on top of the existing flag description?

```
 -h, --hostname string   The GitHub host to use for authentication
```